platform/src/main/java/org/hillview/utils/Utilities.java

```diff
@@ -176,6 +176,14 @@ public static String getHostName() {
         }
     }
 
+    /**
+     * Checks whether the local OS is Windows
+     * 
```

Comment:
no need for this extra empty line. The comment should say: return true if the operating system of the executing program is Windows.